apps/studio/data/sql/queries/get-user.ts

```diff
@@ -0,0 +1,35 @@
+export const getUserSQL = (userId: string) => {
+  const sql = /* SQL */ `
+select
```

Comment:
matches the SQL here
https://github.com/supabase/supabase/blob/master/packages/pg-meta/src/sql/studio/get-users-paginated.ts#L100